libs/logging/src/log_event_ids.ts

```diff
@@ -48,7 +48,11 @@ export enum LogEventId {
   BallotPrinted = 'ballot-printed',
   PrintedBallotReportPrinted = 'printed-ballot-report-printed',
   SmartcardProgramInit = 'smartcard-program-init',
-  SmartcardProgrammed = 'smartcard-programmed',
+  SmartcardProgramComplete = 'smartcard-program-complete',
+  SmartcardUnprogramInit = 'smartcard-unprogram-init',
+  SmartcardUnprogramComplete = 'smartcard-unprogram-complete',
+  // TODO(https://github.com/votingworks/vxsuite/issues/2085):
+  // Remove SmartcardProgrammedOverrideWriteProtection when removing legacy smartcards screen
   SmartcardProgrammedOverrideWriteProtection = 'smartcard-programmed-override-write-protection',
```

Comment:
I won't be using this log event in the new smartcard programming flows, since overriding write protection will now be opaque to the user. Our new flows make it much less likely to accidentally overwrite an admin card, since you have to explicitly unprogram a card before you can reprogram it